server_scripts/glassfish_administer_service.py

```diff
@@ -15,39 +15,42 @@
 
 _PARALLEL_GC = "-XX:-UseParallelGC"
 _PARALLEL_GC_ESC = "-XX\:-UseParallelGC"
+_VERBOSE = False
```

Comment:
Please make this an instance variable for the class, not a global variable

Reply:
Done.

Reply:
This is still an instance variable.